app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    persons_list = [Person(person["name"], person["age"]) for person in people]
+
+    for person in people:
+        person_check = Person.people[person["name"]]
+        if person.get("wife"):
+            person_check.wife = Person.people[person["wife"]]
```

Comment:
The code assumes that the `wife` key in the dictionary corresponds to a name that exists in `Person.people`. Ensure that the input data is validated to prevent `KeyError` if the name does not exist.